index.html

```diff
@@ -15,7 +15,7 @@
 <body>
     <!-- Nav Bar -->
     <nav class="shadow navbar navbar-expand-lg navbar-dark bg-dark">
-        <a class="navbar-brand" href="">QuickSketch</a>
+        <a class="navbar-brand" href="file:///C:/Users/benmr/QuickSketch/QuickSketch/index.html">QuickSketch</a>
```

Comment:
I was about to take a look at the about page, and I like it! you just have to fix the href links for the pull request to be accepted.

Since the files you're linking to are located in the root folder you can simply leave it as

href="index.html"